test/kennel/syncer_test.rb

```diff
@@ -105,9 +105,13 @@ def add_identical
   let(:expected) { [] }
   let(:project_filter) { [] }
   let(:tracking_id_filter) { [] }
+
+  let(:kennel) { Kennel::Engine.new }
+
```

Comment:
nothing else has the newlines 🤷 
```suggestion
  let(:kennel) { Kennel::Engine.new }
```